connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -2744,6 +2744,16 @@ abstract class AvroSuite
       }
     }
   }
+
+  test("SPARK-46990: read an empty file where pastSync returns false at EOF") {
```

Comment:
I am getting error when running this test locally:
```
Caused by: java.util.NoSuchElementException
	at org.apache.avro.file.DataFileStream.next(DataFileStream.java:262)
	at org.apache.avro.file.DataFileStream.next(DataFileStream.java:248)
	at org.apache.spark.sql.avro.AvroUtils$RowReader.hasNextRow(AvroUtils.scala:210)
	at org.apache.spark.sql.avro.AvroUtils$RowReader.hasNextRow$(AvroUtils.scala:198)
	at org.apache.spark.sql.avro.AvroFileFormat$$anon$1.hasNextRow(AvroFileFormat.scala:137)
```

Reply:
This could be related to your environment. The tests pass for me and on CI. Please share more details so I can debug further.

Reply:
It's ok. Probably my env issue. Mark this as resolved.